build_pros.cmake

```diff
@@ -88,9 +89,16 @@ endif()
 
 if(MSVC)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /wd4244 /wd4018 /wd4996 /wd4048 /wd4267 /wd4127")
-    # set(CMAKE_LINK_LIBRARY_FLAG "${CMAKE_LINK_LIBRARY_FLAG} /wd4273")
 elseif(NOT "${CMAKE_C_COMPILER_ID}" MATCHES "^(Apple)?Clang$")
```

Comment:
Why are you excluding clang from these checks??

Reply:
> Why are you excluding clang from these checks??

Because these flags were are GCC-only, and I didn't saw them at Clang (or even, they cause wrong argument warnings at CLang). Do you want use these checks always? I guess, a good idea, why not

Reply:
Well, I think we should _not_ use these warning disablements: 
If there is something to warn related to these, the warning can be
useful. I.e.: just remove them.

Reply:
Then, take the care to fix all these warnings completely, however, MSVC-side warnings were bogus and annoying